### PR TITLE
Add CloudRail

### PR DESCRIPTION
**Pull request by felixkm**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Looks interesting, but seeing how it's just an SDK to a SaaS, it's not a good fit for this list. I realize you have a generous free plan, but from experience free services have a tendency to go away abruptly.

I think https://github.com/ripienaar/free-for-dev would be a better fit ;)

**Comment by felixkm** [comment]
It is a standalone library and not tied to any SaaS. You can download the SDK and use it without ever touching our SaaS offering, nor will any data pass one of our servers. The SaaS is just an optional add-on to be notified about changing APIs etc. So maybe it is a good fit for both lists? :)

**Comment by sindresorhus** [comment]
Oh, ok. That was not clear. I would make that very explicit in the readme.

File: readme.md [hunk 1]
```diff
@@ -621,6 +626,7 @@
 - [passwordless](https://passwordless.net) - Token-based authentication middleware for Express allowing authentication without passwords.
 - [Lockit](https://github.com/zemirco/lockit) - Full featured authentication solution for Express. Supports a variety of databases, predefined routes, email and two-factor authentication.
 - [Grant](https://github.com/simov/grant) - OAuth middleware for Express, Koa, and Hapi.
+- [CloudRail](https://github.com/CloudRail/cloudrail-si-node-sdk) - Unified API for Social Authentication (Facebook, Twitter, Slack, Instagram, ...).
```
Comment (sindresorhus) [inline readme.md]: > Unified API for social authentication (Facebook, Twitter, Slack, Instagram, etc).

**Comment by sindresorhus** [comment]
Happy to merge when the merge conflict and inline feedback is resolved :)

**Comment by felixkm** [comment]
Thanks for the feedback. I'll try to make it more clear. Also updated according to the inline feedback. I'm not sure where the merge conflict is and how to solve it to be honest. Could you help me with that very quick? Thanks!

**Comment by SamVerschueren** [comment]
@felixkm I believe you can click the `Use the command line to resolve conflicts before continuing.` which will "guide" you to the process.

**Comment by sindresorhus** [comment]
Added it for you ;)

**Comment by felixkm** [comment]
Thank you! I didn't have git on the computer I used yesterday and via the GitHub UI I couldn't figure out a way to solve the conflicts.

**Pull request closed without merging**